tst/regression/test_suites/output_hdf5/output_hdf5.py

```diff
@@ -119,12 +119,15 @@ def Analyse(self, parameters):
         if ret_2d != 0 or ret_3d != 0:
             analyze_status = False
 
-        hst_2d = np.genfromtxt("advection_2d.hst")
-        hst_3d = np.genfromtxt("advection_3d.hst")
+        hst_2d = np.genfromtxt("advection_2d.out1.hst")
+        hst_3d = np.genfromtxt("advection_3d.out1.hst")
         ref_results = [
             ["time", 1.0, 1.0],
             ["dt", 1.75781e-03, 3.12500e-03],
             ["total", 7.06177e-02, 1.39160e-02],
+            ["power0", 7.06177e-02, 1.39160e-02],
+            ["power1", 3.88112e-02, 2.59597e-03],
+            ["power2", 2.65948e-02, 7.19427e-04],
```

Comment:
This should be `advected_powers#`, isn't it?
Also, if there are no objection, I'd prefer the indices separated by `_` as we're using that kind of convention in other places already.

Reply:
Yes, the other labels were actually wrong as well (e.g. `total` should have been `total_advected`). I changed those and also added a test of the column headers.

IN ADDITION I also enforced a space between each column so we can parse the labels more easily. In `develop` currently we can get headers like `# [0]=time    [1]=dt   [2]=my_long_var_name[3]=my_other_longvar_name` which can be annoying to parse via something simple like `.split()` in python. I don't think this is a breaking change because whitespace between variable names was already a possibility (for short variable names) but I should mention it.